features/networking/egress-ip.feature

```diff
@@ -136,18 +135,28 @@ Feature: Egress IP related features
     # Patch egress IP to the project twice
     Given I have a project
     And as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
-      | {"egressIPs": ["<%= cb.newip %>"]} |
-    And as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
-      | {"egressIPs": ["<%= cb.valid_ip %>"]} |
+      | {"egressIPs": ["<%= cb.valid_ips[0]%>"]} |
+
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run command on the "<%= node.name%>" node's sdn pod:
+      | bash | -c | ip address show <%= cb.interface %> |
+    Then the step should succeed
+    And the output should contain "<%= cb.valid_ips[0] %>"
+    And the output should not contain "<%= cb.valid_ips[1] %>"
```

Comment:
do we ever have collisions with substrings?  Maybe we should add `/` to the end of the string match to delimit it.

for example
```
valid_ips[0] = 192.168.1.212
valid_ips[1] = 192.168.1.2  
```

will `192.168.1.2` substring match `192.168.1.212`?  

maybe this never happens for some other reason?

Only risk is when the last octet is `.1` and `.2` I guess.

Reply:
Thanks @rbbratta ,  hmmm, this is a kind of corner case but may actually happen.  Updated code use equal to exactly match.